### DISCUSSION TITLE
Low Memory Mode

### DISCUSSION THREAD

**Discussion by tgrieshop** [ravendb/ravendb#15860] [Q&A] [answered]
I have a question for the group related to memory consumption.  We received an alert for low memory on our RavenDB server and noticed we are running in "Low Memory Mode".  I reviewed the RavenDB documentation and cannot find any details on "Low Memory Mode".  Can someone please explain what this is?  Are features not available in this mode?  

![image](https://user-images.githubusercontent.com/25092207/220355200-b0a6b779-dcdb-4620-b2fb-f620b5b2c983.png)

We will be adding additional memory and plan to increase the MaxFreeCommittedMemoryToKeepInMb setting to 1GB but want to make sure we understand the situation before we make these changes.

Thank You

**Comment by garayx** [comment, marked as answer]
Hi 

There is not much impact on the performance, RavenDB is more careful about how it allocate (for example will create smaller indexing batches), and may fail certain operations, such starting of backup task.
The typical reason for that is that you have a big backup running, or long running processes.

To give you more details, there are a number of endpoints that RavenDB expose that can give you more details.
```
/admin/debug/memory/stats
/admin/debug/proc/meminfo
/admin/debug/memory/low-mem-log
```

**Reply by tgrieshop** [reply]
Thanks for the info!
